docs/src/keps/kep-001.rst

```diff
@@ -0,0 +1,172 @@
+Often it makes sense for the prototype implementation to be made available as PR to the Klio repo (making sure to appropriately mark the PR as a ``WIP``).
+
+
+Review and Resolution
+^^^^^^^^^^^^^^^^^^^^^
+
+KEPs are discussed on their respective pull request. The possible paths of the status of KEPs are as follows:
+
+.. figure:: assets/kep_workflow.png
+    :alt: KEP Workflow
+
+    *KEP Workflow*
+
+
+All KEPs should be created as a new pull request against the ``master`` branch with the ``Draft`` status and the `"kep" GitHub tag`_.
+When a status is updated from ``Draft``, the author should update its status in the PR before merging it into ``master``.
+
+Eventually, after discussion, there may be a consensus that the KEP should be accepted – see the next section for details.
+At this point the status becomes ``Accepted``.
+
+Once a KEP has been ``Accepted``, the reference implementation must be completed.
+When the reference implementation is complete and incorporated into the main source code repository, the status will be changed to ``Final``.
+
+To allow gathering of additional design and interface feedback before committing to long term stability for a feature or API, a KEP may also be marked as "Provisional".
+This is short for "Provisionally Accepted", and indicates that the proposal has been accepted for inclusion in the reference implementation, but additional user feedback is needed before the full design can be considered "Final".
+Unlike regular accepted KEPs, provisionally accepted KEPs may still be ``Rejected`` or ``Withdrawn`` even after the related changes have been included in a Klio release.
+
+Wherever possible, it is considered preferable to reduce the scope of a proposal to avoid the need to rely on the "Provisional" status (e.g. by deferring some features to later KEPs), as this status can lead to version compatibility challenges in the wider Klio ecosystem.
+
+A KEP can also be assigned status ``Deferred``.
+The KEP author or a core developer can assign the KEP this status when no progress is being made on the KEP.
+
+A KEP can also be ``Rejected``.
+Perhaps after all is said and done it was not a good idea.
+It is still important to have a record of this fact.
+The ``Withdrawn`` status is similar: it means that the KEP author themselves has decided that the KEP is actually a bad idea, or has accepted that a competing proposal is a better alternative.
+
+When a KEP is ``Accepted``, ``Rejected``, or ``Withdrawn``, the KEP should be updated accordingly.
+In addition to updating the status field, at the very least the ``Resolution`` header should be added with a link to the relevant PR thread.
+
+KEPs can also be ``Superseded`` by a different KEP, rendering the original obsolete.
+The ``Replaced-By`` and ``Replaces`` headers should be added to the original and new KEPs respectively.
+
+Process KEPs may also have a status of ``Active`` if they are never meant to be completed, e.g. KEP 0 (this KEP).
```

Comment:
Does a Process KEP _have_ to go to the `Active` state?  If not, what determines when it does vs just going to `Accepted`?

Similarly, while not explicitly mentioned, it appears that Standards and Informational KEPs cannot become `Active`, only `Accepted`...is that actually the case?

Also I think this means to refer to `KEP 1` (or `KEP 001`?).

Reply:
A KEP does not have to go into the `Active` state. It's only in an `Active` state if it can't conceptually be completed. A Standards Track one (e.g. a feature) can be completed - therefore it goes from `Draft` -> `Accepted` -> dev process -> `Final` (the happy path I guess). A process one like this KEP-001 (or [pep-0001](https://www.python.org/dev/peps/pep-0001/), [pep-0008](https://www.python.org/dev/peps/pep-0008/)) and I'm guessing informational ones ([pep--0247](https://www.python.org/dev/peps/pep-0257/), [pep-8000](https://www.python.org/dev/peps/pep-8000/)) will always remain in an `Active` state, and just go from `Draft` -> `Active` (I guess they can be superseded too).

So yeah, Informational can become `Active`. The Standards track shouldn't really be `Active`, just go from `Draft` -> one of `Accepted`, `Provisional`, `Rejected`, or `Withdrawn`. 

I could add some clarifying language if you think that'd help?

Reply:
Sort of related, I am going to add something like [this summary](https://www.python.org/dev/peps/) to the `index.rst`/landing page of KEPs so we can easily see get an overview of everything (I'll probably add a "status" column too).

Reply:
Ah it looks like the standards track for [Python PEPs](https://www.python.org/dev/peps/#pep-status-key) are the only ones that can have the `Accepted` status.